lib/config/options/index.ts

```diff
@@ -68,6 +68,30 @@ const options: RenovateOptions[] = [
     default: false,
     globalOnly: true,
   },
+  {
+    name: 'mergeConfidenceEndpoint',
```

Comment:
@rarkins Should we add `experimental` or `advancedUse` flags?

Reply:
advancedUse, because most people won't need to use this